azurerm/resource_arm_recovery_services_protected_vm_test.go

```diff
@@ -331,6 +379,209 @@ resource "azurerm_recovery_services_protected_vm" "test" {
+  }
+}
+
+resource "azurerm_public_ip" "test" {
+  name                = "acctest-ip"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  allocation_method   = "Dynamic"
+  domain_name_label   = "acctestip%[1]d"
+}
+
+resource "azurerm_storage_account" "test" {
+  name                     = "acctest%[3]s"
+  location                 = "${azurerm_resource_group.test.location}"
+  resource_group_name      = "${azurerm_resource_group.test.name}"
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+}
+
+resource "azurerm_managed_disk" "test" {
+  name                 = "acctest-datadisk"
+  location             = "${azurerm_resource_group.test.location}"
+  resource_group_name  = "${azurerm_resource_group.test.name}"
+  storage_account_type = "Standard_LRS"
+  create_option        = "Empty"
+  disk_size_gb         = "1023"
+}
+`, rInt, location, rstr[len(rstr)-5:])
+}
+
+// For update backup policy id test
+func testAccAzureRMRecoveryServicesProtectedVm_withVault(rInt int, location string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+resource "azurerm_recovery_services_vault" "test" {
+		name                = "acctest-%[2]d"
+		location            = "${azurerm_resource_group.test.location}"
+		resource_group_name = "${azurerm_resource_group.test.name}"
+		sku                 = "Standard"
+}
+`, testAccAzureRMRecoveryServicesProtectedVm_basePolicyTest(rInt, location), rInt)
+}
+
+// For update backup policy id test
+func testAccAzureRMRecoveryServicesProtectedVm_withFirstPolicy(rInt int, location string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+resource "azurerm_recovery_services_protection_policy_vm" "test" {
+  name                = "acctest-%[2]d"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  recovery_vault_name = "${azurerm_recovery_services_vault.test.name}"
+
+  backup {
+    frequency = "Daily"
+    time      = "23:00"
+  }
+
+  retention_daily {
+    count = 10
+  }
+}
+`, testAccAzureRMRecoveryServicesProtectedVm_withVault(rInt, location), rInt)
+}
+
+// For update backup policy id test
+func testAccAzureRMRecoveryServicesProtectedVm_withSecondPolicy(rInt int, location string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+resource "azurerm_recovery_services_protection_policy_vm" "test_change_backup" {
+  name                = "acctest2-%[2]d"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  recovery_vault_name = "${azurerm_recovery_services_vault.test.name}"
+
+  backup {
+    frequency = "Daily"
+    time      = "23:00"
+  }
+
+  retention_daily {
+    count = 55
+  }
+}
+`, testAccAzureRMRecoveryServicesProtectedVm_withFirstPolicy(rInt, location), rInt)
+}
+
+// For update backup policy id test
+func testAccAzureRMRecoveryServicesProtectedVm_withVM(rInt int, location string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+resource "azurerm_virtual_machine" "test" {
+		name                  = "acctestvm-%[2]d"
```

Comment:
Could we use spaces here for consistency?

Reply:
Converted to spaces